themes/digital.gov/layouts/partials/core/topics-button.html

```diff
@@ -0,0 +1,50 @@
+{{/* Renders a topics button with the name of the topic and has two display options.
+  Option 1 is a styled button with a hover effect and Option 2 is a link followed by the amount of pages that use the topic.
+
+  @param {url} link - Accepts a string path to a topic page
+  @param {string} title - Name of the topic
+  @param {number} tag_count - Number of times the topic tag is used
+  @param {boolean} has_tag_count - Checks if tag_count number is present
+*/}}
+
+{{ $link := .link }}
+{{ $title := .title }}
+{{ $tag_count := .tag_count }}
+{{ $has_tag_count := false }}
+
+{{ if ne $tag_count nil }}
+  {{ $has_tag_count = true }}
+{{ end }}
+
+
+<div
+  class="topics-button {{ if not $has_tag_count }}
+    topics-button--primary
+  {{ else }}
+    topics-button--count
+  {{ end }}"
+>
+  <a
+    class="topics-button__link"
+    href="{{- $link | absURL -}}"
```

Comment:
Should this be relative since they're topics within DigitalGov?

```suggestion
    href="{{- $link | relURL -}}"
```

Reply:
yep good catch... updated to be a relative link